docs/usage.rst

```diff
@@ -763,14 +763,167 @@ As for arrays with different units, for operation between two arrays with units
 Writing Data with Units to Disk
 -------------------------------
 
-Pickles
-*******
+The :mod:`unyt` library has support for serializing data stored in a
+:class:`unyt.unyt_array <unyt.array.unyt_array>` instance to HDF5 files, text
+files, and via the Python pickle protocol. We give brief examples below, but first describe how to handle saving units manually as string metadata.
+
+Dealing with units as strings
+*****************************
+
+If all you want to do is save data to disk in a physical unit or you are working
+in a physical unit system, then you only need to save the unit name as a string
+and treat the array data you are trying to save as a regular numpy array, as in
+this example:
+
+  >>> import numpy as np
+  >>> import os
+  >>> from unyt import cm
+  ...
+  >>> data = [1, 2, 3]*cm
+  >>> np.save('my_data_cm.npy', data)
```

Comment:
Not necessary to do now, but I'm wondering if we ought to have a function that does this for you.

Reply:
Probably worth adding. I’m not sure offhand how extensible the .npy format is.